db/migrations/20230610184101_village_rep.php

```diff
@@ -0,0 +1,26 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Migration\AbstractMigration;
+
+final class VillageRep extends AbstractMigration
+{
+    /**
+     * Change Method.
+     *
+     * Write your reversible migrations using this method.
+     *
+     * More information on writing migrations is available here:
+     * https://book.cakephp.org/phinx/0/en/migrations.html#the-change-method
+     *
+     * Remember to call "create()" or "update()" and NOT "save()" when working
+     * with the Table class.
+     */
+    public function change(): void
```

Comment:
Can you split this into `up` and `down` methods? The `change` method is for Phinx reversible migration syntax, for manual SQL queries I think we need to define both ourselves.